### PR TITLE
Add jamshale as maintainer

### PR DESCRIPTION
**Pull request by jamshale**
I would like to become a maintainer of this repo. Mainly so I can merge approved PR's but also to help review code and help make decisions on the direction of this repo.

I plan on working on it for an extended period of time and believe I meet all the requirements to become a maintainer.

@swcurran @WadeBarnes

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Maintainers are assigned the following scopes in this repository:
 | dbluhm          | Daniel Bluhm     |            |      |            |       | Indicio PBC         |
 | dhh1128         | Daniel Hardman   | Maintainer |      |            |       | Provenant           |
 | ianco           | Ian Costanzo     |            |      |            |       | Anonymous Systems   |
+| jamshale        | Jamie Hale       | Maintainer |      |            |       |                     |
```
Comment (swcurran) [inline MAINTAINERS.md:29]: Please add BC Gov as the organization.  Thanks!

Comment (jamshale) [reply MAINTAINERS.md]: Updated. I rebased into one commit.

**Review by swcurran — APPROVED** [review]

**Comment by swcurran** [comment]
@ryjones — could you please make the necessary Hyperledger Team update to implement this change?

**Comment by ryjones** [comment]
@swcurran done; however, you also had the power to do this.

**Comment by swcurran** [comment]
Oh — sorry about that.  I’ll make sure I try to do that myself. I didn’t realize it.

Thanks!

**Comment by ryjones** [comment]
it's only when they aren't members of the org that I need to do something.

**Pull request merged**